angrmanagement/ui/widgets/qjobs.py

```diff
@@ -0,0 +1,302 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import qtawesome as qta
+from PySide6.QtCore import QSize, Qt
+from PySide6.QtWidgets import (
+    QAbstractItemView,
+    QHBoxLayout,
+    QHeaderView,
+    QLabel,
+    QProgressBar,
+    QPushButton,
+    QSizePolicy,
+    QTableWidget,
+    QTableWidgetItem,
+    QVBoxLayout,
+    QWidget,
+)
+
+if TYPE_CHECKING:
+    from angrmanagement.data.jobs import job
```

Comment:
Should be `Job` not `job` right? Once that is fixed, consider renaming variables `the_job` to just `job`.